administrator/modules/mod_quickicon/mod_quickicon.xml

```diff
@@ -30,6 +30,13 @@
 					description="MOD_QUICKICON_GROUP_DESC"
 					default="mod_quickicon"
 				/>
+				<field
+					name="header_icon"
+					type="text"
+					label="MOD_QUICKICON_HEADER_ICON_LABEL"
+					description="MOD_QUICKICON_HEADER_ICON_DESC"
```

Comment:
see previous comment about the _DESC string

Reply:
I'll us e a hint. I think users mus know that there a fontawesome icon is expected.